ff/bw6-assembly/modadd768.S

```diff
@@ -0,0 +1,181 @@
+// void modadd768(const uint64_t x[12], const uint64_t y[12], const uint64_t m[13], uint64_t z[12])
```

Comment:
Was there any automated process for how you got this assembly, or was it hand-written?

If there was an automated process, which was then edited for optimizations, could you share it please?

If it was hand-written, would it be possible to add some comments for how you derived whats going on?
(And also what features its doing, that leads it to be faster than native in expectation)

Reply:
I believe that there was automatic process, and we eventually want to convert all the asm code to using .M, I think, as I really dislike inline ASM and also its the wrong usecase, since inline asm is meant for very short segments of code that one does indeed intend to be "inlined".

@kobigurk should know more details on the automated process.